R/fertile.R

```diff
@@ -186,7 +186,18 @@ proj_analyze_pkgs <- function(path = ".") {
 
 generate_script <- function(pkg_name, vector = c()) {
 
-  new_line <- sprintf("install.packages('%s')", pkg_name)
+  # check if package is available on CRAN
+  pkg_on_cran <- !as.logical(available::available_on_cran((pkg_name)))
+  pkg_on_github <- !as.logical(available::available_on_github((pkg_name)))[1]
+
+  if(pkg_on_cran == TRUE){
+    new_line <- sprintf("install.packages('%s')", pkg_name)
+  }else if(pkg_on_github == TRUE){
+    new_line <- sprintf("#Package '%s' is located on GitHub. Find its author and install using devtools::install_github()", pkg_name)
```

Comment:
Can you change this from `devtools` to `remotes`?

Reply:
Yes, just did!